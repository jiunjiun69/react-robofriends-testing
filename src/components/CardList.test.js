import React from 'react';
import CardList from './CardList';
import { create } from "react-test-renderer";

const filteredRobots = [{
    id: 1,
    name: 'Leanne Graham',
    username: 'Bret',
    email: 'Sincere@april.biz'
  }]

const cardList = create(<CardList robots={filteredRobots}/>);
it('renders without crashing', () => {
    expect(cardList.toJSON()).toMatchSnapshot(); // check if a component renders correctly
});