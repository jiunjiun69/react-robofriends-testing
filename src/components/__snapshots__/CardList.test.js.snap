// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    className="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/1?size=200x200"
    />
    <div>
      <h2>
        Leanne Graham
      </h2>
      <p>
        Sincere@april.biz
      </p>
    </div>
  </div>
</div>
`;
